sdcm/nemesis.py

```diff
@@ -3693,10 +3686,10 @@ def decommission_post_action():
 
             ParallelObject(objects=[trigger, watcher], timeout=timeout).call_objects()
             if new_node := decommission_post_action():
-                self.wait_node_fully_start(new_node)
+                self.cluster.wait_node_fully_start(new_node)
```

Comment:
so, in the case i started describing, here you would call it `new_node.wait_fully_start`